src/components/containers/popover/PopoverLink.js

```diff
@@ -65,7 +67,8 @@ class PopoverLink extends React.Component {
           }}
         >
           <PopoverButton
-            aria-haspopup="dialog"
+            aria-haspopup="true"
```

Comment:
Aa, okay. I don't remember them talking about the "dialog" option, but I did find some stuff on it through the [aria 1.1 testable statements](https://www.w3.org/wiki/ARIA_1.1_Testable_Statements#aria-haspopup) which works with role="button". 
Which might be part of our problem now that I think about it, since we're using role="tooltip" for popovers and not "button" with aria-haspopup="dialog".
But I think it's still suggested to use role="dialog" with aria-labelledby instead, at least from what I have written in my notes.